WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -96,6 +131,15 @@ public SendViewModel(Wallet wallet)
 
 			PasteCommand = ReactiveCommand.CreateFromTask(async () => await OnPasteAsync());
 			AutoPasteCommand = ReactiveCommand.CreateFromTask(async () => await OnAutoPasteAsync());
+			QRCommand = ReactiveCommand.Create(async () =>
+			{
+				if (!_qrReader.IsRunning)
+				{
+					IsCameraLoadingAnimationVisible = true;
+					IsQrPanelVisible = true;
+					await _qrReader.StartScanningAsync();
```

Comment:
In case of error, the IsCameraLoadingAnimationVisible stays visible.

Reply:
This was resolved by a new dialog